bug(cli): server list -v will panic

https://github.com/tensorchord/openmodelz/blob/main/mdz/pkg/cmd/server_list.go#L103

```
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
github.com/tensorchord/openmodelz/mdz/pkg/cmd.labelsString(0xc0005c56b8?)
	/home/runner/work/openmodelz/openmodelz/mdz/pkg/cmd/server_list.go:103 +0x165
github.com/tensorchord/openmodelz/mdz/pkg/cmd.commandServerList(0x1bfed40, {0xc0004676d0?, 0x1?, 0x1?})
	/home/runner/work/openmodelz/openmodelz/mdz/pkg/cmd/server_list.go:72 +0x885
github.com/spf13/cobra.(*Command).execute(0x1bfed40, {0xc0004676b0, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x1c00ce0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/tensorchord/openmodelz/mdz/pkg/cmd.Execute()
	/home/runner/work/openmodelz/openmodelz/mdz/pkg/cmd/root.go:46 +0x25
main.main()
	/home/runner/work/openmodelz/openmodelz/mdz/cmd/mdz/main.go:9 +0x17
```

--- a/mdz/pkg/cmd/server_list.go
+++ b/mdz/pkg/cmd/server_list.go
@@ -100,6 +100,9 @@ func labelsString(labels map[string]string) string {
 	for k, v := range labels {
 		res += fmt.Sprintf("%s=%s\n", k, v)
 	}
+	if len(res) == 0 {
+		return res
+	}
 	return res[:len(res)-1]
 }
 

